### DISCUSSION TITLE
Add a Now Playing sensor

### DISCUSSION THREAD

**Discussion by jimmyz5150** [music-assistant/support#3191] [Feature requests and ideas]
I'd like to say the short time coming across this, it's got a future! Love I can finally play from Plex inside HA. Not to mention adding in others.
- Can you figure out a way to add a sensor that would at minimum show a Now Playing [Plex or Spotify or Tune-In, etc..]. I have my music setup to show if it is idle or not. Thought it would be a cool addition. See picture
![Screenshot 2024-11-16 171753](https://github.com/user-attachments/assets/0b20ddf9-f00f-48a7-a2b7-60e9efa2c42b)

**Comment by OzGav** [comment]
You can use the [get_queue](https://music-assistant.io/faq/get_queue/) action and build your own

**Reply by jimmyz5150** [reply]
hmm ok I'll look into thx!

**Reply by jimmyz5150** [reply]
Hey OzGav just wondering are there suppose to be entity states or something for MA. All I see under Dev-States is update.music_assistant_server_update. MA newbie here lol

**Reply by OzGav** [reply]
You need to go to HA SETTINGS >> INTEGRATIONS >> MUSIC ASSISTANT to see what entities have been created.

**Reply by jimmyz5150** [reply]
I didn't have anything in Integrations, just in add-ons. Went to HACS and had to download and then I finally have Integrations

**Reply by OzGav** [reply]
OK. Review all of our docs for more information

**Comment by OzGav** [comment]
Closing as relates to old versions
